library/std/src/os/unix/fs.rs

```diff
@@ -1064,7 +1064,6 @@ pub fn lchown<P: AsRef<Path>>(dir: P, uid: Option<u32>, gid: Option<u32>) -> io:
 /// }
 /// ```
 #[stable(feature = "unix_chroot", since = "1.56.0")]
-#[cfg(not(any(target_os = "fuchsia", target_os = "vxworks")))]
```

Comment:
Leave this cfg for Fuchsia, [it is probably in fact *not* a Unix](https://github.com/rust-lang/rust/issues/58590).

Reply:
I realize I am being somewhat inconsistent here but I would prefer to deal with the Fuchsia question separately.

Reply:
Sure, will do.